src/olympia/ratings/tests/test_helpers.py

```diff
@@ -30,7 +30,11 @@ def test_stars_details_page(self):
 
     def test_stars_max(self):
         doc = pq(self.render('{{ num|stars }}', {'num': 5.3}))
-        assert doc.attr('class') == 'stars stars-5'
+        assert doc.attr('class') == 'stars stars-5-0'
+    
+    def test_stars_css(self):
+        doc = pq(self.render('{{ num|stars }}', {'num': 3.3}))
+        assert doc.attr('class') == 'stars stars-3.5'
```

Comment:
You haven't replaced the `.` like we discussed so the `class` is wrong - it should be `stars-3-5` here if it worked properly.

Reply:
I just modified, and forgot to replace only, my bad.
Will do it right away